lib/joke.rb

```diff
@@ -0,0 +1,34 @@
+class Joke
+  attr_reader :id, :setup, :punchline
+
+  def initialize(id, setup, punchline)
+    @id = id
+    @setup = setup
+    @punchline = punchline
+  end
+
+
+
```

Comment:
Looks great! Usually we wouldn't see this big white space between the end of initialize and the end of the class.

Reply:
Thank-you; I am still struggling with how it is supposed to look.